backends/gdx-backends-gwt/src/com/badlogic/gdx/backends/gwt/DefaultGwtInput.java

```diff
@@ -1223,6 +1233,11 @@ protected int keyForCode (int keyCode, int location) {
 	private static final int KEY_F24 = 135;
 	private static final int KEY_NUM_LOCK = 144;
 	private static final int KEY_SCROLL_LOCK = 145;
+	private static final int KEY_AUDIO_VOLUME_DOWN = 174;
+	private static final int KEY_AUDIO_VOLUME_UP = 175;
+	private static final int KEY_MEDIA_TRACK_NEXT = 176;
+	private static final int KEY_MEDIA_TRACK_PREVIOUS = 177;
+	private static final int KEY_MEDIA_PLAY_PAUSE = 179;
```

Comment:
It came to my mind that we are totally missing STOP, just as a remark.

Reply:
That's a key that isn't on my keyboard, but I can add it

Reply:
My keyboard has one and it is reported as `178`. The rest of the PR looks good to me!